scripts/install/jellyfin.sh

```diff
@@ -40,6 +34,17 @@ if [[ -n $active ]]; then
     fi
 fi
 
+#
+## Get the path to gpg or install it
+GNUPG=$(which gpg)
```

Comment:
gnupg2 is a global dependency of swizzin. You can safely assume every user has this installed already.